skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
It looks like rseqc may have been using that code without a more explicit attribution. 

@mortonjt, would it be possible to include the relevant license in the licenses [directory](https://github.com/biocore/scikit-bio/tree/master/licenses)?